org.eclipse.xtext.xbase.tests/longrunning/src/org/eclipse/xtext/xbase/tests/typesystem/TimedBatchTypeResolverTest.xtend

```diff
@@ -60,7 +61,7 @@ class TimedBatchTypeResolverTest extends AbstractBatchTypeResolverTest {
 class TypeResolverPerformanceTest extends BatchTypeResolverTest {
 	
 	@Rule
-	public val timeout = new Timeout(100) // TODO improve - aim at something like 100
+	public val timeout = new Timeout(100, TimeUnit.MILLISECONDS) // TODO improve - aim at something like 100
```

Comment:
comment makes no sense

Reply:
The test class is ignored because the timout is too strict. But yes, the comment is not too helpful.